src/BaseDraw.php

```diff
@@ -516,7 +516,7 @@ public function modulo($Value1, $Value2)
             $Factor = $Factor * 10;
         }
 
-        return ($Value1 * $Factor) % ($Value2 * $Factor);
+        return floor($Value1 * $Factor) % floor($Value2 * $Factor);
```

Comment:
Can you please tell me how do the values here change after adding these?

Reply:
They actually shouldn't change at all. The floor is, what PHP will do in background. The modulo operation (`%`) works with integers only. And when casting a float to integer PHP simply cuts off the fraction digits.
See this for a simple demonstration: https://3v4l.org/EehJN